Src/Newtonsoft.Json.UnityConverters/PartialConverter/ValuesArray.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Newtonsoft.Json.UnityConverters
+{
+    public readonly struct ValuesArray<TInner> : IReadOnlyList<TInner>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Implement 'IEquatable<T>' in value type 'ValuesArray'.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5420094)